apis/metadata_validation_test.go

```diff
@@ -106,3 +106,50 @@ func TestValidateObjectMetadata(t *testing.T) {
 		})
 	}
 }
+
+func TestServiceAnnotationUpdate(t *testing.T) {
+	const (
+		u1 = "oveja@knative.dev"
+		u2 = "cabra@knative.dev"
+		groupName = "pkg.knative.dev"
+	)
+	tests := []struct {
+		name          string
+		old           interface{}
+		new           interface{}
+		oldAnnotation map[string]string
+		newAnnotation map[string]string
+		want          *FieldError
+	}{{
+		name: "update creator annotation",
+		old:  nil,
```

Comment:
All of the test cases have `old` and `new` as nil. What about where there are spec changes and the lastModifier should be updated. Can we add that test case as well?

Reply:
Added test case related to spec and /lastModifier changes.